plugin/webdevicons.vim

```diff
@@ -96,23 +96,24 @@ function s:getDistro()
     return s:distro
   endif
 
-  let s:lsb = system('lsb_release -i')
-
-  if s:lsb =~# 'Arch'
-    let s:distro = ''
-  elseif s:lsb =~# 'Ubuntu'
-    let s:distro = ''
-  elseif s:lsb =~# 'Cent'
-    let s:distro = ''
-  elseif s:lsb =~# 'Debian'
-    let s:distro = ''
-  elseif s:lsb =~# 'Dock'
-    let s:distro = ''
-  else
-    let s:distro = ''
+  if executable('lsb_release -i')
```

Comment:
Executable need only exec target file.
Remove `-i`

Reply:
@tsuyoshicho
Thanks for code review.
I fixed